app/src/main/java/com/mapbox/mapboxsdk/plugins/testapp/activity/offline/OfflineRegionDetailActivity.java

```diff
@@ -188,36 +188,36 @@ public void onFabClick(View view) {
   }
 
   @Override
-  public void onCreate(OfflineDownloadOptions offlineDownload) {
-    Timber.e("OfflineDownloadOptions created %s", offlineDownload.hashCode());
+  public void onCreate(DownloadOptions offlineDownload) {
```

Comment:
Maybe change the parameter name to `downloadOptions` or something without the word offline in it if the class name is no longer going to have the word `Offline` in it...